motionplan/plannerOptions.go

```diff
@@ -1,19 +1,26 @@
 package motionplan
 
 import (
+	"fmt"
 	"math"
 	"runtime"
 
+	"github.com/edaniels/golog"
+	pb "go.viam.com/api/service/motion/v1"
 	"gonum.org/v1/gonum/floats"
+
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
 )
 
 // default values for planning options.
 const (
 	// max linear deviation from straight-line between start and goal, in mm.
 	defaultLinearDeviation = 0.1
```

Comment:
I didn't see a default linear deviation in the scope. Where is it?

Reply:
Changing the existing defaults was not part of the scope, aside from the change in how orientation deviation was defined. This is the same default for linear deviation that has been used since linear motion became the default for all arm.MoveToPosition calls last April.

Reply:
Since these constraints were in the extras before, any defaults should have been part of this scope, since we shouldn't assume they will stay the same. We should add in these defaults to the scope and email scope reviewers.

Reply:
Added this to the scope doc:

> Assumption: all constraint implementation defaults will remain the same, aside from Orientation (whose units are changing from R3AA norm to # degrees)

Note that the constraint system and its production use by other parts of RDK significantly predate their inclusion in the motion service via `extra`. `extra` is not the only place they are used, and to change their defaults means changing user-facing observed behavior of other systems that are not the motion service.

Reply:
Thank you!

Reply:
I am OK with the caveat these values DO NOT CHANGE (outside of the requisite orientation adjustment), and this is an informative gesture. The values we already implement have been significantly battle tested and are run against continuously in CI. We don't want to be in the business of changing these things arbitrarily, and we have a lot of experience with the current values.